backend/config/.env_example

```diff
@@ -1,3 +1,4 @@
 DB_PASS=Password
```

Comment:
What's the purpose of this file?

Reply:
it has environmental variables

Reply:
it has environment variables

Reply:
It would be cool to reflect that in the name as well, so you wouldn't have to look inside, maybe environment-variables-example or something

Reply:
".env" it isnt name,it type
thats why .env_example cool,u can delete "_example" part,put there your values and work with it